api/services/Reddit.js

```diff
@@ -8,14 +8,15 @@ exports.data = {
   clientID: sails.config.reddit.clientID,
   clientIDSecret: sails.config.reddit.clientIDSecret,
   redirectURL: sails.config.reddit.redirectURL,
-  adminRefreshToken: sails.config.reddit.adminRefreshToken
+  adminRefreshToken: sails.config.reddit.adminRefreshToken,
+  userAgent: "Webpage:hq.porygon.co:v" + sails.config.version;
```

Comment:
I changed this to something a bit more descriptive, but we can change it back if necessary